README.md

```diff
@@ -162,24 +162,23 @@ The following FAILS because it's missing api tags
 class TestClass {}
 ```
 
-### Rule: api-method-should-specify-api-operation
+### Rule: api-method-should-specify-api-response
 
-If you have an api method like @Get() you should specify the return status code (and type!) by using @ApiOkResponse and the other expected responses. I often leave out 400s and 500s because it's kind of assumed but they should be used if the return type changes!
+If you have an api method like @Get() you should specify the return status code (and type!) by using @ApiResponse and the other expected responses. I often leave out 400s and 500s because it's kind of assumed but they should be used if the return type changes!
 
 This PASSES
 
 ```ts
 class TestClass {
     @Get()
-    @ApiOkResponse({type: String, isArray: true})
-    @ApiBadRequestResponse({description: "Bad Request"})
```

Comment:
i think it's better to show that multiple response types are ok in the documentation. It's not just for `ApiResponse`. The rule allows the following

```
            "ApiResponse",
            "ApiOkResponse",
            "ApiCreatedResponse",
            "ApiAcceptedResponse",
            "ApiNoContentResponse",
            "ApiMovedPermanentlyResponse",
            "ApiFoundResponse",
            "ApiBadRequestResponse",
            "ApiUnauthorizedResponse",
            "ApiTooManyRequestsResponse",
            "ApiNotFoundResponse",
            "ApiInternalServerErrorResponse",
            "ApiBadGatewayResponse",
            "ApiConflictResponse",
            "ApiForbiddenResponse",
            "ApiGatewayTimeoutResponse",
            "ApiGoneResponse",
            "ApiMethodNotAllowedResponse",
            "ApiNotAcceptableResponse",
            "ApiNotImplementedResponse",
            "ApiPreconditionFailedResponse",
            "ApiPayloadTooLargeResponse",
            "ApiRequestTimeoutResponse",
            "ApiServiceUnavailableResponse",
            "ApiUnprocessableEntityResponse",
            "ApiUnsupportedMediaTypeResponse",
            "ApiDefaultResponse",
```